### PR TITLE
Common: Migrate logging to fmt

### PR DESCRIPTION
**Pull request by lioncash**
Continues the migration of our code over to the fmt logger.

### CODE REVIEW

File: Source/Core/Common/HttpRequest.cpp [hunk 1]
```diff
@@ -221,8 +221,8 @@ HttpRequest::Response HttpRequest::Impl::Fetch(const std::string& url, Method me
   curl_easy_getinfo(m_curl.get(), CURLINFO_RESPONSE_CODE, &response_code);
   if (response_code != 200)
   {
-    ERROR_LOG(COMMON, "Failed to %s %s: server replied with code %li and body\n\x1b[0m%.*s", type,
-              url.c_str(), response_code, static_cast<int>(buffer.size()), buffer.data());
+    ERROR_LOG_FMT(COMMON, "Failed to {} {}: server replied with code {} and body\n\x1b[0m{:.{}}",
```
Comment (BhaaLseN) [inline Source/Core/Common/HttpRequest.cpp:224]: Just wondering, why does this one have an ANSI reset code in it?

**Review by BhaaLseN — APPROVED** [review]

File: Source/Core/Common/HttpRequest.cpp (hunk 1 above)
Comment (lioncash) [reply Source/Core/Common/HttpRequest.cpp:224]: I legitimately don't know

Comment (leoetlino) [reply Source/Core/Common/HttpRequest.cpp:224]: For the record:

```
[23:44:56] <@Bh44L> leoetlino: do you remember why that log message in 9187 has an ANSI reset code in it?
[23:45:16] <@Bh44L> came from 5599
[23:47:05] <@leoetlino> uh, I think it was to make the response body stand out more in stderr output
[23:48:01] <@leoetlino> it's not strictly required, but it makes it easier to see where the body begins and ends when you have several ERROR_LOGs in a row
[23:56:40] <@Bh44L> oh, right, log goes to console as well
[23:56:42] <@Bh44L> ¯\_(ツ)_/¯
```

**Review by leoetlino — APPROVED** [review]

**Comment by brujo5** [comment]
Hello, sorry for the inconvenience, could you check this please?

https://forums.dolphin-emu.org/Thread-dolphin-5-0-12919-cause-dolphin-emulator-close?action=lastpost

File: Source/Core/Common/FileUtil.cpp [hunk 2]
```diff
@@ -346,13 +343,13 @@ bool RenameSync(const std::string& srcFilename, const std::string& destFilename)
...
 {
-  INFO_LOG(COMMON, "Copy: %s --> %s", source_path.c_str(), destination_path.c_str());
+  INFO_LOG_FMT(COMMON, "Copy: {} --> {}", source_path, destination_path);
 #ifdef _WIN32
   if (CopyFile(UTF8ToTStr(source_path).c_str(), UTF8ToTStr(destination_path).c_str(), FALSE))
     return true;
 
-  ERROR_LOG(COMMON, "Copy: failed %s --> %s: %s", source_path.c_str(), destination_path.c_str(),
-            GetLastErrorString().c_str());
+  ERROR_LOG_FMT(COMMON, "Copy: failed %s --> %s: %s", source_path, destination_path,
```
Comment (JosJuice) [inline Source/Core/Common/FileUtil.cpp:351]: This looks like the problem.

**Comment by brujo5** [comment]
update to 5.0-12923 but dolphin 
Captura de pantalla (26)
emulator still close it.

**Pull request merged**